pkg/daemon/ceph/client/info.go

```diff
@@ -95,7 +95,7 @@ func AdminClusterInfo(ctx context.Context, namespace, name string) *ClusterInfo
 	return &ClusterInfo{
 		Namespace: namespace,
 		CephCred: CephCred{
-			Username: AdminUsername,
+			Username: NonOperatorAdminUsername,
```

Comment:
@travisn since you are planning to look, I found this method is being in other places too which has `client.admin`.

Reply:
I really thought changing this to the `OperatorAdminUsername` would fix the tests, but still something else to investigate...